config/csv/config.go

```diff
@@ -33,6 +33,31 @@ func EmptyNull(emptyNull bool) ConfigFunc {
 	}
 }
 
+// MissingColumnNameAliasString set configs to rename empty column name with given string
+//  ,COL2 			// col   COL2
+// a,1.5   				a     1.5
+// b,2.25  				b    2.25
+// c,3.0`				c       3
+
+func MissingColumnNameAlias(MissingColumnNameAlias string) ConfigFunc {
+	return func(c *Config) {
+		c.MissingColumnNameAlias = MissingColumnNameAlias
+	}
+}
+
+// AddColumnIndexToDuplicateColumns configures if column names have duplicates
+// column index will be added to the column name
+// COL,COL 			// COL0   COL1
```

Comment:
I think we can skip the inline examples here and instead create proper example tests covering the usage.

Reply:
I need help with this, What should be the function names in examples_test.go file. for examples.